server/bin/pbench-index.py

```diff
@@ -161,19 +203,19 @@ def main(options, name):
         "INCOMING", INCOMING_rp, idxctx.logger
     )
     if not INCOMING_path:
-        res = 3
+        res = error_codes["BAD_CFG"].value
```

Comment:
Here instead of assigning a value to `res`, should we assign an ErrorCode object itself to it something like `res = error_codes["BAD_CFG]`. That way we can later check if the res is ignorable or what is the value of res when we are returning it. So on line 222 we can return it as `return res.value` instead of `return res`. It will also help on line 476 where we need to check whether `res` or `tb_res` is ignorable or not by just doing `if res.ignorable or tb_res.ignorable`

Reply:
got the idea now. Thanks!